Source/SwiftLintBuiltInRules/Rules/Style/OpeningBraceRule.swift

```diff
@@ -37,6 +43,9 @@ private extension OpeningBraceRule {
 
         override func visitPost(_ node: EnumDeclSyntax) {
             let body = node.memberBlock
+            if isIgnoredMultilineConditionStatement(body, keywordToken: node.enumKeyword) {
+                return
+            }
```

Comment:
An _enum_ definition could have multiline protocol adherence, but I"m not aware of a condition list. So, is this needed? 

The same question applies to all other applications of this logic that do not have an associated condition list that precedes the opening brace.

Reply:
The last version of the code I pushed applied this change for all syntax options so that we could see the differences in some open-source projects in the report that SwiftLintBot provides us. I had a long absence from this PR and didn't push a new version nor notified about this, sorry about that. Thanks for your contribution!